source/GameAction.h

```diff
@@ -87,6 +89,9 @@ class GameAction {
 	// When this action is performed, the missions with these names fail.
 	std::set<std::string> fail;
 
+	const Planet *teleportPlanet = nullptr;
+	bool flagshipOnly = false;
```

Comment:
`flagshipOnly` doesn't seem like the best name. Only the flagship for what? I'd say `onlyRelocateFlagship` or something like that.

Reply:
renamed